### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

### CODE REVIEW

**Review by Pierre-Sassoulas — APPROVED** [review]
Amazing ! Thank you :)

File: pylint/epylint.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def lint(filename, options=()):
             if line.startswith("No config file found"):
                 continue
 
-            # modify the file name thats output to reverse the path traversal we made
+            # modify the file name that's output to reverse the path traversal we made
```
Comment (DanielNoord) [inline pylint/epylint.py:128]:
```suggestion
            # modify the file name that's put out to reverse the path traversal we made
```

**Review by DanielNoord — APPROVED** [review]
Sadly I recognised some of my own mistakes in there.. 😅 

Thanks a lot 👍

**Comment by Pierre-Sassoulas** [comment]
Congratulation on becoming a pylint contributor :tada:

**Pull request merged**